File: pyramidi/pyramidi.js
const pyramid = function (layers) {
  const output = [];
  const blankSpace = layers * 2 - 2;

  for (let i = 0; i < layers; i++) {
    let str = "*";
    for (let y = 0; y < i; y++) str += "**";

    const spaces = blankSpace / 2 - i;
    str = " ".repeat(spaces) + str + " ".repeat(spaces);
    output.push(str);
  }

  return output;
};

const amount = prompt("Amount of layers?");
const result = pyramid(amount);
for (const x of result) console.log(x);

File: pyramidi/pyramidi.py
def pyramid(layers):
    stars = layers * 2 - 1

    output = []
    for i in range(layers):
        star = "*"
        for x in range(i):
            star += "**"

        layer = star.center(stars)
        output.append(layer)

    return output


layers = int(input("Layers: "))
result = pyramid(layers)

for entry in result:
    print(entry)
